apps/web/src/hooks/workflowChannelPreferences/useUpdateWorkflowChannelPreferences.ts

```diff
@@ -17,6 +18,7 @@ export const useUpdateWorkflowChannelPreferences = (
     WorkflowChannelPreferences
   >((data) => api.upsertPreferences(workflowId, data), {
     onSuccess: () => {
+      onSuccess();
```

Comment:
```suggestion
      onSuccess?.();
```
Could be done like that, instead of  setting empty default value ()=>{}